frontend/src/components/Spinner.scss

```diff
@@ -0,0 +1,61 @@
+// Taken from here https://projects.lukehaas.me/css-loaders/
```

Comment:
It is just for example, right?

Reply:
yes, just random filler to show it's uploading